docs/DEPLOYMENT.md

```diff
@@ -155,13 +159,15 @@ cd ui
 docker build -t <image> . && docker push <image>
 ```
 
+eg. `quay.io/<username>/ui`
```

Comment:
Why do we need this here ?? :thinking:

Reply:
its an example

Reply:
And also mention if one pushes the image to quay, he needs to make that image public